src/components/app/App.js

```diff
@@ -6,10 +6,12 @@ import Footer from "./Footer/Footer"
 import './App.css';
 
 const App = () => (
-    <div className={"main"}>
-        <Route path="/:term?" component={GlossaryPage}/>
-        <Footer className={"main__footer"}/>
-    </div>
+  <section>
```

Comment:
Because the main content is the rendered by route, so i think that the two more appropriate tags, in this scenario would be, article and section, but dont make sense article, once, this is not will reusable

Reply:
I think that `section` it's not the appropriate tag, look:

>Do not use the `<section>` element as a generic container; this is what `<div>` is for, especially when the sectioning is only for styling purposes. A rule of thumb is that a section should logically appear in the outline of a document.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
What do you suggest?

Reply:
and right now?

Reply:
Fanny I totally understand why we should avoid using `div` all the time, but I think this is one of the cases which `div` is the appropriate tag to be used.